justfile

```diff
@@ -10,18 +10,25 @@ set dotenv-path := "."
 set dotenv-filename := "./scripts/rust.env"
 
 # enable to debug just recipes
+
 debug_justfile := "false"
 [private]
 dpdk_sys_commit := shell("source ./scripts/dpdk-sys.env && echo $DPDK_SYS_COMMIT")
 [private]
 _just_debuggable_ := if debug_justfile == "true" { "set -x" } else { "" }
+
 # the tripple to compile for
+
 target := "x86_64-unknown-linux-gnu"
+
 # cargo build profile to use
+
 profile := "debug"
 [private]
 _container_repo := "ghcr.io/githedgehog/dataplane"
+
 # the rust channel to use (choose stable, beta, or nightly)
+
 rust := "stable"
 [private]
 _dpdk_sys_container_repo := "ghcr.io/githedgehog/dpdk-sys"
```

Comment:
Having a blank line all around comments, but not between other declarations, looks weird and not easy to read. Do we really want that? Or should we at least use blank lines between all declarations for the paragraph, in that case?

Reply:
This is just how the stupid just format function works.  I also hate it.